source/index.html

```diff
@@ -2,11 +2,160 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
+    <link rel="icon" type="image/png" sizes="32x32" href="../source/img/icon/favicon.png">
+    <link rel="stylesheet" href="../source/css/">
     <title>HTML Academy: Кэт энерджи</title>
   </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+    <h1 class="visually hidden">CAT ENERGY</h1>
+    <header class="main__header">
+      <a class="header__logo logo__link" href="#">
+        <img class="logo-link__img" width="33" height="37" alt="Логотип с котом." src="../source/img/icon/logo_mobile.svg">
+      </a>
+      <a class="header__logo-main logo-main__link" href="#">
+        <img class="logo-main__img" width="101" height="18" alt="Логотип магазина" src="../source/img/icon/logo_main.svg">
+      </a>
+      <button class="main-header__button-burger" type="button"></button>
+      <div class="main-header__popover popover__menu">
+        <button class="menu__button-close" type="button">
+          <span class="visually-hidden">Закрыть.</span>
+        </button>
+        <div class="menu__content">
+          <ul class="menu__content-list">
+            <li class="menu__content-item">
+              <a class="menu__content-link">Главная</a>
+            </li>
+            <li class="menu__content-item">
+              <a class="menu__content-link" href="catalog.html">Каталог продукции</a>
+            </li>
+            <li class="menu__content-item">
+              <a class="menu__content-link" href="form.html">Подбор программы</a>
+            </li>
+          </ul>
+        </div>
+      </div>
```

Comment:
1. Нет подписи
2. **По желанию** можно добавить aria атрибуты. В `aria-controls` пишется `id` элемента за отображение которого отвечает кнопка. 
3. Навигацию стоит обернуть в `nav`. 
4. Второй див внутри особо без надобности
5. Кнопка закрыть не нужно, потому что предполагается, что меняется кнопка открытия
```suggestion
      <button class="main-header__button-burger" type="button" aria-expanded="false" aria-controls="main-nav" >Открыть меню</button>
      <nav class="main-header__popover popover__menu" id="main-nav">
          <ul class="menu__content-list">
            <li class="menu__content-item">
              <a class="menu__content-link">Главная</a>
            </li>
            <li class="menu__content-item">
              <a class="menu__content-link" href="catalog.html">Каталог продукции</a>
            </li>
            <li class="menu__content-item">
              <a class="menu__content-link" href="form.html">Подбор программы</a>
            </li>
          </ul>
      </nav>
```

Reply:
Не очень поняла: зачем эта подпись, что она даёт...